src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Calibration/HeadsetCalibration.cs

```diff
@@ -125,7 +139,7 @@ private void ProcessQRCodeUpdate()
             {
                 updatedMarkerIds.Add(marker.Key);
                 float size = 0;
-                if (qrCodeMarkerDetector.TryGetMarkerSize(marker.Key, out size))
+                if (markerDetector.TryGetMarkerSize(marker.Key, out size))
                 {
                     var qrCodeTopLeftPosition = CalcTopLeftFromCenter(marker.Value.Position, marker.Value.Rotation, size);

```

Comment:
>qrCodeTopLeftPosition [](start = 24, length = 21)

we should probably change the naming here so that this isn't declared as a qr code. Although this works it will likely generate in the future.

Reply:
I have renamed all related identifiers, except for those where a new name is required for the distinction between the two marker types (e.g. [debug visuals](https://github.com/microsoft/MixedReality-SpectatorView/pull/359/files?file-filters%5B%5D=.cs#diff-7a16755085586350abce224c5cfea0a0L53-L54)). Do you have a preference on such a naming?
A suggestion from my side would be `detectedMarker...` and `calculatedMarker...`

Reply:
detectedMarker and calculatedMarker sounds good to me :)